_posts/2025-05-19-journey-to-find-replacements-java-security-manager.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "Journey to find a replacement to the Java Security Manager in OpenSearch 3.0"
```

Comment:
Wonder if `create` or `propose` is better than `find` for the title.

Reply:
I agree. "find" is appropriate here since the blog delves into the lengthy process of identifying a suitable replacement. It also goes into creating the replacement so you could argue to make it `find and create`, but I think it also works with `find` alone.